platform/windows/detect.py

```diff
@@ -71,6 +71,7 @@ def get_opts():
         BoolVariable("use_mingw", "Use the Mingw compiler, even if MSVC is installed. Only used on Windows.", False),
         BoolVariable("use_llvm", "Use the LLVM compiler", False),
         BoolVariable("use_thinlto", "Use ThinLTO", False),
+        BoolVariable("use_static_cpp", "When using MinGW, link libgcc, libstdc++ and libpthread statically", True),
```

Comment:
The description no longer fits now that it also controls MSVC runtime linking.

Reply:
Correct. 

Does this look good? I am open to suggestions:

> Link MinGW/MSVC runtime libraries statically

Reply:
Sounds good, maybe:
>Link MinGW/MSVC C++ runtime libraries statically

Reply:
Done!